src/utils/constants.ts

```diff
@@ -0,0 +1,3 @@
+export class Constants {
+  static readonly passwordRegex = /((?=.*d)|(?=.*W+))(?![.\n])(?=.*[A-Z])(?=.*[a-z]).*$/;
```

Comment:
Do you need class for it?

Reply:
I used class because I can't store regex in enum. Since I don't plan to have many constants I can maybe make passwordRegex env variable. It will introduce more setup flexibility for other users too

Reply:
I decided to just store it in the separate file without class and changed the regex itself as well to more readable format